openshift-kube-apiserver/admission/customresourcevalidation/route/validation_wrapper.go

```diff
@@ -0,0 +1,99 @@
+package route
+
+import (
+	"fmt"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/library-go/pkg/apiserver/admission/admissionrestconfig"
+	authorizationv1client "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubernetes/openshift-kube-apiserver/admission/customresourcevalidation"
+)
+
+type validateCustomResourceWithClient struct {
+	admission.ValidationInterface
+
+	secretsGetter             corev1client.SecretsGetter
+	sarGetter                 authorizationv1client.SubjectAccessReviewsGetter
+	routeValidationOptsGetter RouteValidationOptionGetter
+}
+
+func NewValidateRoute() (admission.Interface, error) {
+	ret := &validateCustomResourceWithClient{}
+
+	delegate, err := customresourcevalidation.NewValidator(
+		map[schema.GroupResource]bool{
+			configv1.GroupVersion.WithResource("routes").GroupResource(): true,
+		},
+		map[schema.GroupVersionKind]customresourcevalidation.ObjectValidator{
+			configv1.GroupVersion.WithKind("Route"): routeV1{
```

Comment:
Why is this `configv1.GroupVersion` instead of `routev1.GroupVersion`?

Reply:
that must be a typo, fixed it!

Reply:
It seems like this plugin is severely lacking in test coverage if existing tests didn't catch this.  What's the consequence of having the wrong API group?  Does the plugin effectively not enforce any validation, as it's registered for the wrong API?  Did you test this manually to verify that it actually enforces validation?